### DISCUSSION TITLE
Skinning out "ready" sound in multi lobbies

### DISCUSSION THREAD

**Discussion by yeahirai** [ppy/osu#31112] [Ideas]
I've been having a lot of fun on multi lobbies in lazer but there is this one thing that keeps bothering me whenever i join a lobby with too many players, the sound that plays when someone press the ready button. People seem to think that this is a "hurry up" button instead, where instead of just pressing it to indicate that they are ready to play the map they use it to spam the other people in the lobby with this sound to put pressure on whoever didn't press it already. This is a problem that increase whenever a lobby gets bigger and all the other 18 people of the lobby decides to spam the button. It's so obnoxious that i have looked into ways to skin this sound out of the game by replacing it with a empty sound but either i'm bad at searching or this can't be skinned yet, for that reason i'm adding this request here so maybe someone can enlighten me on how to do that if its already possible.

**Comment by peppy** [comment]
It kind of *is* a hurry up button. We may look at making it less obnoxious in the future though.

cc @nekodex
